funnel/assets/js/util.js

```diff
@@ -38,8 +38,7 @@ export const Utils = {
     );
   },
   smoothScroll() {
-    $('a.js-smooth-scroll').on('click', function clickHandler(event) {
-      event.preventDefault();
+    $('a.js-smooth-scroll').on('click', function clickHandler() {
       Utils.animateScrollTo($(this.hash).offset().top);
```

Comment:
What does this do?

Reply:
Add the hash to the url for tabs. For ex: `#about`, `#talks`, `#venue` on the project page